data-access-analysis-tools/index-rsp-warnings.rst

```diff
@@ -12,98 +12,60 @@
-
-
-
-Stuff That We Know Does Not Work
---------------------------------
-
-*(Here is space for a list of known issues and stuff that doesn’t work in , e.g., the portal. Especially the portal. To avoid people thinking “This is too hard to figure out” when really it’s just a feature that’s not done.)*
-
-ADQL functionality that does not yet work: 
-AREA
-BOX
-COORDSYS
-COORD1
-COORD2
-INTERSECTS
-
-The only coordinate system currently supported is 'ICRS'.
+- Notebooks: We are planning on providing a filespace that you can access from your personal device (eg laptop) so you can use your favourite editor/IDE to write code in your local environment and save it in a way that makes it visible in your RSP notebook environment
+- Batch: Opportunities for parallelized/batch/non-interactive computation are not currently available
+- Qserv: User tables are not yet available
+- Qserv: TAP/ADQL Queries using some keywords (AREA, BOX, COORDSYS, COORD1, COORD2 and INTERSECTS) are not yet supported
+- Portal: Many improvements are planned, including the ability to start a query in the portal and access it from your notebook
+- Authentication and Authorisation: You will be able to create and manage groups to allow you to share data with specific RSP users
+- API: Virtual Observatory image services are not yet available
+- API: VOSpace service is not yet available
+- ConsolidatedDB: Access to observatory metadata is not yet available
+- Bulk Download Service: Not offered
 
+Warnings and admonishments
+---------------------------
 
-MLG Did Not Know The Category
------------------------------
+Our `Acceptable Use Policy <https://data-dev.lsst.cloud/terms>`_ applies: We’re giving you access to Rubin Observatory systems so you can do science with the data we provide or otherwise further the mission of the observatory. You can lose your access if you misuse our resources, interfere with other users, or otherwise do anything that would bring the observatory into disrepute.
 
-The DP0.1 repo at IDF is one older version of the /repo/dc2 repo at NCSA.  New updates in NCSA’s repo are not ported to IDF.  Note that the goals of the two repos are different: the IDF’s repo is frozen like a data release; the NCSA’s repo is evolving to meet developer need. 
-(HFC: This is only to avoid confusion for those delegates who are also in the Stack Club or have access to the LSST infrastructure at NCSA.)
+One of the main goals of the Data Preview program is to allow us to assess user interactions with our services. Bypassing use of our services (such as attempting to bulk download data or trying to defeat the Butler APIs) violate the spirit of the exercise.
```

Comment:
"violates"

Reply:
Before saying it's okay to remove this warning, I wanted to get @MelissaGraham 's opinions on whether there will be many users that use both NCSA's RSP and IDF's RSP.   The core message I want here is that the Butler repo at NCSA and the Butler repo at IDF are just different, so don't expect them to be identical.

Reply:
Per Melissa on Slack: 

>  I suspect it’s not worth writing a specific set of “what’s changed” just for Stack Club members, even though there might be quite a few. Or, if you did, you could post it in #stack-club maybe?

Reply:
Posted Hsinfang's wording on stack-club per Melissa's suggestion.